ide/db.sql.editor/src/org/netbeans/modules/db/sql/editor/resources/layer.xml

```diff
@@ -58,7 +58,7 @@
 
                 <folder name="FoldManager">
                     <file name="CustomFoldManager.instance">
-                	<attr name="position" intvalue="1200"/>
+                        <attr name="position" intvalue="1200"/>
```

Comment:
Please remove the unnecessary whitespace changes.

Reply:
Looking at the code all the "attr" around this seem to line up.  

If I'm not mistaken it looks like previously it was a tab now is 4 spaces as is currently present.

When I formatted the context of the "folder" tag it also does not appear to change. 

Does my "spacing formatting" preferences need to be set in some way to conform to a specific spacing standard for xml?

What should the whitespace be set to?

Reply:
The reasoning is simple: Don't change things you don't need to change. Changing whitespace in places, that don't need an update just messes diffs. There are reasons to do this, this is not one of them.

In general: Check the surrounding code - independent of any documentation new code should align with local conventions. If in doubt, I use default netbeans settings.

Reply:
Okay understand that changing spaces is a pain in update reviews and should tried to avoid it whenever possible.  

But all that said, comparing the change here on that line with the apache/netbeans master branch version of the file...it looks like the master had a tab on that line but all others used spaces for indenting the lines.

![image](https://user-images.githubusercontent.com/370894/114189733-225d5300-9910-11eb-99a1-e4a01172af93.png)

Looking through the rest of the file I don't see tabs used anyplace else.

I can revert that line back and add the tab back if that is the preference.

Reply:
Okay...if I did my git correctly, I've removed the layer.xml from changes in the commit and the Resource.properties file which appear to not have any significant changes and unneeded.  Is this good for comments against these two files?

Reply:
Yes the changes are gone (git diff locally or the github view are helpful).